code/modules/species/outsider/random.dm

```diff
@@ -15,6 +15,11 @@
 	base_color  = RANDOM_RGB
 	. = ..()
 
+/decl/bodytype/alium/Initialize()
```

Comment:
`/decl/bodytype/alium/Initialize()` already exists directly above this, please combine these overrides.